modules/apps/27-interchain-accounts/controller/types/msgs_test.go

```diff
@@ -0,0 +1,76 @@
+package types_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+	icatypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/types"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
+)
+
+var (
+	testAccAddress     = "cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs"
+	testMetadataString = icatypes.NewDefaultMetadataString(ibctesting.FirstConnectionID, ibctesting.FirstConnectionID)
+)
+
+func TestMsgRegisterAccountValidateBasic(t *testing.T) {
+	var msg *types.MsgRegisterAccount
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success",
+			func() {},
+			true,
+		},
```

Comment:
want to add tests for different forms of the version (just to ensure we aren't doing version validation in the msg validate basic)

Reply:
Done!